content/6_Advanced/Treaps.mdx

```diff
@@ -326,6 +328,215 @@ With split and merge, how do we implement the additional operations?
   in lazy propogation). When splitting or merging, we push these tags downwards
   and perform the corresponding operation.
 
+### Implementation
```

Comment:
(optional)

rename to problem name - implementation to be consistent with the cut and paste header